mk/main.mk

```diff
@@ -58,9 +57,7 @@ plugins: build-plugins
 cross: build-x
 
 clean: coverage-clean build-clean
-test: dco fmt vet test-short
+test: dco fmt test-short vet
 validate: dco fmt vet lint test-short test-long
-install:
-	cp ./bin/docker-machine* /usr/local/bin/
```

Comment:
Unfortunately, this is not portable, it also ignores the output PREFIX. Better out for now until this is done properly (honor DESTDIR etc), or we have other install mechanisms.